akka-remote/src/main/scala/akka/remote/artery/FlightRecorder.scala

```diff
@@ -291,13 +300,18 @@ private[akka] class FlightRecorder(val fileChannel: FileChannel) extends AtomicB
       loFreqLogs.markSnapshot(previousLog)
       alertLogs.markSnapshot(previousLog)
       fileChannel.force(true)
-      set(false)
+      snapshotLatch.countDown()
+      set(Running)
```

Comment:
`compareAndSet(SnapshotInProgress, ...)` ? 
e.g. if it was changed to `ShutDown`